api/backup.go

```diff
@@ -18,7 +18,38 @@ func (s *Server) BackupTargetList(w http.ResponseWriter, req *http.Request) erro
 	if err != nil {
 		return errors.Wrap(err, "failed to list backup targets")
 	}
-	apiContext.Write(toBackupTargetCollection(backupTargets))
+	apiContext.Write(toBackupTargetCollection(backupTargets, apiContext))
+	return nil
+}
+
+func (s *Server) SyncBackupTarget(w http.ResponseWriter, req *http.Request) error {
+	var input SyncBackupTarget
+
+	apiContext := api.GetApiContext(req)
+	if err := apiContext.Read(&input); err != nil {
+		return err
+	}
+
+	btName := mux.Vars(req)["btName"]
```

Comment:
nit: replace `btName` with `backupTargetName`

Reply:
I assume this PR will apply to multiple backup targets feature.
Should we
1. synchronize all backup targets if `btName` is empty
2. set `btName` to `default` is if `btName` is empty
3. error out if `btName` is empty
Which one is better?

If we choose 2 or 3, can we sync all backup targes?

Reply:
Modified to `backupTargetName`.
I took 2 and 3 because python clients, it is not easy to update backup targets/volumes with an empty name.

I added two APIs, one is to synchronize all backup targets and another is to synchronize all backup volumes.